barretenberg/cpp/src/barretenberg/relations/databus_lookup_relation.hpp

```diff
@@ -57,6 +57,17 @@ template <typename FF_> class DatabusLookupRelationImpl {
         LENGTH  // log-derivative lookup argument subrelation
     };
 
+    /**
+     * @brief Upper bound on total degrees of sub-relations considered as polynomials in witnesses.
+     *
+     */
+    static constexpr std::array<size_t, NUM_BUS_COLUMNS * 2> SUBRELATION_WITNESS_DEGREES{
```

Comment:
please explain here the -1 and in the other places, it really helps the reader in a few months